templates/artist/includes/artist_detail_overview.html

```diff
@@ -47,8 +47,10 @@ <h4 class="investing">${{ campaign.amount_raised|intcomma }}</h4>
                         <button class="button login-box" data-open="login-modal">LOGIN</button>
                     {% endif %}
                 {% else %}
-                    <h3 class="campaign-funded">FUNDED!</h3>
-                    <p><em>Campaign ended {{ campaign.end_datetime|date:'F jS' }}</em></p>
+                    {% if campaign.percentage_funded == 100 %}
+                        <h3 class="campaign-funded">FUNDED!</h3>
+                        <p><em>Campaign ended {{ campaign.end_datetime|date:'F jS' }}</em></p>
```

Comment:
I think it should still show the campaign end date if the campaign is no longer open.